tool-plugins/intellij/src/main/gen/io/ballerina/plugins/idea/psi/BallerinaCheckPanicExpression.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we update the year? Need to do this in the template.

Reply:
I've already changed the template and works fine when adding new classes. But not being applied for the generated files :/ Strange

Reply:
Did you try after deleting all the generated files?

Reply:
Yeah I did. Tried with restarting the IDE as well.

Reply:
I think you might have edited a wrong template. Can you let me know what is the template you edited?

Reply:
You need to update the following file and regenerate the classes - https://github.com/ballerina-platform/ballerina-lang/blob/master/tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/grammar/CopyrightHeader.txt

Reply:
Oh my bad :/ I tried changing the file header in IDEA file templates.

Fixed :) Thanks for pointing out the fix.